executor/brie.go

```diff
@@ -46,6 +46,13 @@ import (
 	"github.com/pingcap/tidb/util/sqlexec"
 )
 
+const clearInterval = 10 * time.Minute
+
+var outdatedDuration = types.Duration{
```

Comment:
Can it be a const?
```suggestion
const outdatedDuration = types.Duration{
```

Reply:
No. Go doesn't support struct constants. https://golang.org/ref/spec#Constants